### PR TITLE
Restrain Potential

### PR DESCRIPTION
**Pull request by Nick-Chaos**
Lio may add an extra potential term to Hamiltonian for penalty the distance between specified pairs of atoms. Details about implementaion and uses are in lio/docs/manual/

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,12 +1,12 @@
 .PHONY: all clean hooks
 
 all:
-	cd g2g; make;
+#	cd g2g; make;
```
Comment (ramirezfranciscof) [inline Makefile]: This should not be commented, right?

File: Makefile [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 	cd lioamber; make;
 	cd liosolo; make;
 
 clean:
-	cd g2g; make clean;
+#	cd g2g; make clean;
```
Comment (ramirezfranciscof) [inline Makefile]: This should not be commented, right?

File: Makefile (hunk 1 above)
Comment (Nick-Chaos) [inline Makefile]: No, my bad

File: Makefile (hunk 2 above)
Comment (Nick-Chaos) [inline Makefile]: No, my bad

**Pull request merged**